### PR TITLE
Add recommends to deb package

### PR DESCRIPTION
**Pull request by Theosakamg**
Add to debian package recommends for powershell

### CODE REVIEW

File: stdeb.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [colcon-powershell]
 Depends3: python3-colcon-core (>= 0.3.0)
+Recommends3: powershell
```
Comment (dirk-thomas) [inline stdeb.cfg]: I don't think `powershell` is a valid Debian package in any of the targeted Ubuntu distros. Can you provide a link to the Ubuntu package if there is one?

Comment (Theosakamg) [reply stdeb.cfg]: `Powershell` require extra repository
https://docs.microsoft.com/fr-fr/powershell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6#ubuntu-1604

Comment (dirk-thomas) [reply stdeb.cfg]: Then this dependency can't be specified here since it wouldn't be satisfiable if the user hasn't added this repo.

**Comment by Theosakamg** [comment]
This PR no longer needs to exist so I'll close it.

**Pull request closed without merging**